the_well/benchmark/trainer/training.py

```diff
@@ -225,15 +242,15 @@ def rollout_model(self, model, batch, formatter, train=True):
             )
         y_preds = []
         for i in range(rollout_steps):
-            if not train:
-                moving_batch = self.normalize(moving_batch)
+            # NOTE: This is a quick fix so we can make datamodule behavior consistent. Revisit this next release (MM).
+            if i > 0 and not train:
+                moving_batch, _ = self.normalize(moving_batch)
```

Comment:
Since this PR is interacting with code that is not included in this PR, I just want to make sure we are extra clear in terms what we are doing here.

I assume the implementation in 
https://github.com/PolymathicAI/the_well/blob/6cd3c44ef832855a5abae87d555bf0f0f52b1fa7/the_well/data/datasets.py#L618

means when the dataset is sampled given normalization flag is normalized. Does this not means we are normalizing the batch again?